src/exabgp/reactor/api/command/neighbor.py

```diff
@@ -73,99 +63,98 @@ class Neighbor(object):
     summary_template = '%-15s %-7s %9s %-12s %10d %10d'
 
     @classmethod
-    def as_dict(cls, answer):
-        up = answer['duration']
-
-        formated = {
-            'state': 'up' if up else 'down',
-            'duration': answer['duration'] if up else answer['down'],
-            'fsm': answer['state'],
-            'local': {
-                'capabilities': {},
-                'families': {},
-                'add-path': {},
-            },
-            'peer': {
-                'capabilities': {},
-                'families': {},
-                'add-path': {},
-            },
-            'messages': {'sent': {}, 'received': {}},
-            'capabilities': [],
-            'families': [],
-            'add-path': {},
-        }
-
-        for (a, s), (l, p, aps, apr) in answer['families'].items():
-            k = '%s %s' % (a, s)
-            formated['local']['families'][k] = l
-            formated['peer']['families'][k] = p
-            formated['local']['add-path'][k] = aps
-            formated['peer']['add-path'][k] = apr
-            if l and p:
-                formated['families'].append(k)
-            formated['add-path'][k] = _addpath(aps, apr)
-
-        for k, (l, p) in answer['capabilities'].items():
-            formated['local']['capabilities'][k] = l
-            formated['peer']['capabilities'][k] = p
-            if l and p:
-                formated['capabilities'].append(k)
-
-        for k, (s, r) in answer['messages'].items():
-            formated['messages']['sent'][k] = s
-            formated['messages']['received'][k] = r
-
-        formated['local']['address'] = answer['local-address']
-        formated['local']['as'] = answer['local-as']
-        formated['local']['id'] = answer['local-id']
-        formated['local']['hold'] = answer['local-hold']
-
-        formated['peer']['address'] = answer['peer-address']
-        formated['peer']['as'] = answer['peer-as']
-        formated['peer']['id'] = answer['peer-id']
-        formated['peer']['hold'] = answer['peer-hold']
-
-        return formated
-
-    @classmethod
-    def extensive(cls, answer):
+    def extensive(cls, answer, output_format='text'):
```

Comment:
extensive is unlikely to be right as a @classmethod, it is a normal one class function.

Reply:
One for me to look into if the previous code was already a @classmethod

Reply:
However, using an output_format as a string, not a boolean or an "enum", is wrong. String should not be given magical if it is possible to compare them.